photon-client/src/main/scala/com/linkedin/photon/ml/data/avro/ResponsePredictionFieldNames.scala

```diff
@@ -17,12 +17,7 @@ package com.linkedin.photon.ml.data.avro
 /**
  * Response prediction format fields names.
  */
-object ResponsePredictionFieldNames extends FieldNames {
-  val features: String = "features"
-  val name: String = "name"
-  val term: String = "term"
-  val value: String = "value"
-  val response: String = "response"
-  val offset: String = "offset"
-  val weight: String = "weight"
-}
+object ResponsePredictionFieldNames extends AvroFieldNames {
+
+  override val RESPONSE: String = "response" // do we really need a different name?
```

Comment:
The main crux of the argument is that the prevailing style as of today in the codebase is to not use inline comments, me not liking them is just additional context. Our style should be consistent throughout, and for things that don't affect the correctness of the code, it usually comes down to the personal preferences of the main contributors, as well as the historic impact of those who were contributors before. As far as I know, up to the present time that has not included inline comments.

I think this is specific issue is one of personal taste entirely - IMO correctly formatted comment blocks don't disturb the flow of the code, but rather help group it into finite chunks.

As for the substance of the comment: then make it a TODO. The current text is ambiguous, and in addition it's kind of "playing dumb". We know exactly why we need to have `AvroFieldNames` derived as such into two classes with different "response" field names. We can document it, or we can not - but we shouldn't leave comments like this one wondering why we're doing it and if we should.